apps/challenges/models.py

```diff
@@ -568,6 +568,16 @@ class ChallengeEvaluationCluster(TimeStampedModel):
         max_length=512, null=True, blank=True
     )
     route_table_id = models.CharField(max_length=512, null=True, blank=True)
+    efs_security_group_id = models.CharField(
```

Comment:
I think these will go Challenge Phase model as one environment corresponds to one `phase`. What do you think?

Reply:
We can instead maintain a dict of EFS meta data in `ChallengeEvaluationCluster` model. Right now meta of all components is in the `ChallengeEvaluationCluster` model

Reply:
yes, but `efs_id` will go into the challenge_phase, right?

Reply:
Yeah, we can store it in `ChallengePhase` model